pkgs/development/python-modules/safe-pysha3/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "safe-pysha3";
+  version = "1.0.3";
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Kd+lb9t5ney50BvdfbwPKAb0Ro0sKW+DtuN9hlMZF8I=";
+  };
+
```

```suggestion
  };

  nativeCheckInputs = [
    unittestCheckHook
  ];
```

We always run the tests for python packages, since they break very easily and we need to notice that.